src/bun.js/bindings/headers-handwritten.h

```diff
@@ -138,14 +139,9 @@ const ZigStackFrameCode ZigStackFrameCodeWasm = 5;
 const ZigStackFrameCode ZigStackFrameCodeConstructor = 6;
 
 typedef struct ZigStackFramePosition {
-    int32_t source_offset;
-    int32_t line;
-    int32_t line_start;
-    int32_t line_stop;
-    int32_t column_start;
-    int32_t column_stop;
-    int32_t expression_start;
-    int32_t expression_stop;
+    WTF::OrdinalNumber line;
```

Comment:
Is it safe to use `WTF::OrdinalNumber` here? It won't be initialized as an C++ `WTF::OrdinalNumber` in Zig. I think this should be stored as an int32_t and then a method could be added to convert it to a WTF::OrdinalNumber.

Reply:
yeah lets do this